SPECS/forkexecd.spec

```diff
@@ -37,26 +38,25 @@ ocaml setup.ml -build
 mkdir -p %{buildroot}/%{_libdir}/ocaml
 export OCAMLFIND_DESTDIR=%{buildroot}/%{_libdir}/ocaml
 ocaml setup.ml -install
-mkdir -p %{buildroot}/%{_sbindir}
-install fe_main.native %{buildroot}/%{_sbindir}/forkexecd
-install fe_cli.native %{buildroot}/%{_sbindir}/forkexecd-cli
-mkdir -p %{buildroot}/%{_sysconfdir}/init.d
-install -m 0755 forkexecd-init %{buildroot}%{_sysconfdir}/init.d/forkexecd
-
+%{__install} -D -m 0755 fe_main.native %{buildroot}%{_sbindir}/forkexecd
+%{__install} -D -m 0755 fe_cli.native %{buildroot}%{_sbindir}/forkexecd-cli
+%{__install} -D -m 0644 %{SOURCE1} %{buildroot}%{_unitdir}/forkexecd.service
+%{__install} -D -m 0644 %{SOURCE2} %{buildroot}%{_sysconfdir}/sysconfig/forkexecd
 
 %files
 %{_sbindir}/forkexecd
 %{_sbindir}/forkexecd-cli
-%{_sysconfdir}/init.d/forkexecd
+%{_unitdir}/forkexecd.service
+%{_sysconfdir}/sysconfig/forkexecd
```

Comment:
Yes, you'll see as you go through the other pull-requests I've changed these to %config(noreplace). This one slipped through the net... Bad me.